src/components/drawing/index.js

```diff
@@ -875,11 +926,28 @@ drawing.makeTester = function() {
  * @return {object}: a plain object containing the width, height, left, right,
  *   top, and bottom of `node`
  */
-drawing.savedBBoxes = {};
-var savedBBoxesCount = 0;
-var maxSavedBBoxes = 10000;
+var MAX_SAVED_BBOXES = 10000;
```

Comment:
I suggest using a power of two for this constant e.g. 8192 or 4096 for faster get/set results.